include/ipr/interface

```diff
@@ -198,6 +203,7 @@ namespace ipr {
    // -- result of trinary expression constructor constants --
    // --------------------------------------------------------
    struct New;                   // new-expression              new (p) T(v)
+   struct Coerce;                // implicit conversions not promotion or demotion
```

Comment:
"not promotion, nor demotion"

Reply:
This does not only define implicit conversions. I updated this to define generalized conversions and added more details on the node's block comment.